### PR TITLE
adding missing transform/transformSchema

### PR DESCRIPTION
**Pull request by GoEddie**
When I implemented CountVectorizerModel I missed off transform/transformSchema and without transform it is basically useless. This adds Transform (and TransformSchema) to CountVectorizerModel.

### CODE REVIEW

File: src/csharp/Microsoft.Spark.E2ETest/IpcTests/ML/Feature/CountVectorizerModelTests.cs [hunk 1]
```diff
@@ -28,6 +29,9 @@ public CountVectorizerModelTests(SparkFixture fixture) : base(fixture)
         [Fact]
         public void TestCountVectorizerModel()
         {
+            DataFrame input = _spark.Sql("SELECT array('hello', 'I', 'AM', 'a', 'string', 'TO', " +
+                                         "'TOKENIZE') as input from range(100)");
```
Comment (suhsteve) [inline src/csharp/Microsoft.Spark.E2ETest/IpcTests/ML/Feature/CountVectorizerModelTests.cs:33]:
```suggestion
                "'TOKENIZE') as input from range(100)");
```

File: src/csharp/Microsoft.Spark/ML/Feature/CountVectorizerModel.cs [hunk 2]
```diff
@@ -162,6 +164,35 @@ internal CountVectorizerModel(JvmObjectReference jvmObject) : base(jvmObject)
...
+        public StructType TransformSchema(StructType value) => 
+            new StructType(
+                (JvmObjectReference)_jvmObject.Invoke(
+                    "transformSchema", 
+                    DataType.FromJson(_jvmObject.Jvm, value.Json)));
+        
+        /// <summary>
+        /// Converts a DataFrame with a text document to a sparse vector of token counts.
+        /// </summary>
+        /// <param name="document"><see cref="DataFrame"/> to transform</param>
```
Comment (Niharikadutta) [inline src/csharp/Microsoft.Spark/ML/Feature/CountVectorizerModel.cs:192]: Missing return type

**Review by Niharikadutta — COMMENTED** [review]
One minor nit but otherwise LGTM. Thanks @GoEddie !

**Review by suhsteve — APPROVED** [review]

**Review by imback82 — APPROVED** [review]
LGTM, thanks @GoEddie!

**Pull request merged**